### ISSUE TITLE
Chaos Dwarf District Seat capstones over count and stop working.

### ISSUE THREAD

**Issue by Alexios-Komnenos** [chadvandy/cbfm_wh3#943]
**Describe the bug**
The 3 capstone seats, Mutator, Infernatus, and Procurator, sometimes will over count your number of seats and potentially overflow, depriving you of the buff entirely.

**Expected behavior**
The scaling buff should correspond with your seat count, instead of over counting

**To Reproduce**
I am not entirely sure what causes it, though I believe it to because of AI stealing seats. In the save file attached, I already have the full industry bonus despite lacking 1 seat (so I should be at -18% construction cost instead of -20%) and if Astragoth is confederated, the industry bonus is lost. Additionally, the sorcery bonus appears to work, but if you fill the tree, after confederating the 2nd to last sorcery seat will double count and take you from 24 to 30, and the final seat will overflow and remove the WoM bonus as well. All 3 categories in this save file are broken, and the AI stole at least 1 seat from me in each category.

**Screenshots**
![image](https://user-images.githubusercontent.com/99524979/232715104-61f265a6-f06c-453e-b870-3cfa02c33db0.png)
This screenshot is before confederating Astragoth, as you can see I have the industry buff at full strength.

![image](https://user-images.githubusercontent.com/99524979/232711829-ee4df862-d04e-4f2d-8faa-840f3a268220.png)
This is the same turn, after hitting the confederate button. I no longer have the Industry bonus, and it is impossible to get it back.
**Saves**
[District Seat Bug Save.zip](https://github.com/chadvandy/cbfm_wh3/files/11259380/District.Seat.Bug.Save.zip)

**Comment by Alexios-Komnenos** [comment]
Apologies, I attempted to copy my screenshots from discord, that I sent earlier in the day to some friends. Discord being slow and buggy didn't copy my image when I clicked it, and Github does not have an image preview that I'm aware of, and so it copied a pathfinder meme I had on my clipboard instead. Since I can't edit the text I am just going to post this here. Here is the correct image 
![image](https://user-images.githubusercontent.com/99524979/232714748-fcbcc04a-fed9-4640-b88e-80228a35e4fb.png)
Again, really sorry about that.
